botorch/models/likelihoods/pairwise.py

```diff
@@ -21,7 +21,20 @@
 
 
 class PairwiseLikelihood(Likelihood, ABC):
-    """Pairwise likelihood base class for pairwise preference GP (e.g., PairwiseGP)"""
+    """
```

Comment:
```suggestion
    r"""
```

Reply:
PEP 257:
> For consistency, always use """triple double quotes""" around docstrings. Use r"""raw triple double quotes""" if you use any backslashes in your docstrings. 

The CI is now enforcing this because this PR enabled the D301 lint, which checks that any docstring with a backslash in it is raw. I just tried adding a backslash to this docstring and running `flake8` and it did complain.